ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -0,0 +1,59 @@
+build_root:
+  image_stream_tag:
+    name: ci-tools-build-root
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: prow/Dockerfile
+  to: ocp-qe-perfscale-ci-rosa-4.12-nightly
+promotion:
+  namespace: ci
+  tag: latest
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: ocp-qe-perfscale-ci-tests-rosa-cluster-density
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
+    test:
+    - ref: openshift-qe-cluster-density
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-cluster-density-v2
+  cron: 0 12 * * 5
+  steps:
+    cluster_profile: aws-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
+    test:
+    - ref: openshift-qe-cluster-density-v2
+    workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-node-density
+  cron: 0 12 * * 5
```

Comment:
Should we space out when the tests run? Currently they are all at the same time.

Reply:
Last time I spoke to Roshni, she said this is fine. We can probably rotate the timing between 4.12, 4.13 and Hypershift Vs Rosa instead of each test here. 

But if we want to still keep different cadence, May be we can add next set of workloads to be run at different cadence and keep these 3 as is.

Reply:
yeh just a different time/day with the same cadence for the tests between all different test types and versions would be good